projects/adxrs290-pmdz/src.mk

```diff
@@ -29,7 +29,8 @@ INCS += $(INCLUDE)/no_os_delay.h        \
         $(INCLUDE)/no_os_util.h         \
         $(INCLUDE)/no_os_units.h        \
         $(INCLUDE)/no_os_alloc.h        \
-        $(INCLUDE)/no_os_mutex.h
+        $(INCLUDE)/no_os_mutex.h         \
+        $(INCLUDE)/no_os_pwm.h
```

Comment:
is this needed?

Reply:
The adxrs290-pmdz project has the TIM module enabled in the .ioc file. Hence added the PWM drivers here as we are making a check for the HAL_TIM_MODULE_ENABLED in the stm32_spi.c driver